.travis.yml

```diff
@@ -84,6 +84,7 @@ script:
     fi
   - coala --non-interactive
   - python setup.py docs
+  - man $VIRTUAL_ENV/man/man1/coala.1 > /dev/null 2>&1
```

Comment:
for missing manpages, I see exit code 16, so this looks good.

alternatively/additionally we could check the output for some string we know will exist if it is our manpage.

I suggest you move this to .ci/check_man.sh , and also get it working under Circle CI (which should be green, while Travis is red atm.

Reply:
Thank you for review!
Circle CI test is added now, check itself moved to `.ci/check_man.sh`, output check added (check if `https://coala.io` exists in output)